packages/components/src/components/label/label.tsx

```diff
@@ -17,9 +32,9 @@ const available_modes = [
 
```

Comment:
Since available_modes have same items as type of mode, will this approach work?


const available_modes = ['adjustment', 'default','success','warn', 'danger', 'info', 'transfer', 'default-invert',     'success-invert', warn-invert'] as const;
type mode = typeof available_modes[number];

https://stackoverflow.com/questions/44497388/typescript-array-to-string-literal-type